app/views/users/index.html.erb

```diff
@@ -1,7 +1,3 @@
 <div>
-  <% @users.each do |user| %>
-    <ul>
-      <%= link_to user.name, user %>
-    </ul>
-  <% end %>
+ <%= render @users %>
```

Comment:
This list of users should not include the `current_user`. At the moment, it does! [`OPTIONAL`]